### PR TITLE
2023.1: Backport Cumulus NVUE support

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

File: networking_generic_switch/devices/netmiko_devices/cumulus.py [hunk 1]
```diff
@@ -87,3 +87,79 @@ class Cumulus(netmiko_devices.NetmikoSwitch):
...
+    ]
+
+    PLUG_PORT_TO_NETWORK = [
+        'nv set interface {port} bridge domain br_default access '
+        '{segmentation_id}',
+    ]
+
+    DELETE_PORT = [
+        'nv unset interface {port} bridge domain br_default access '
+        '{segmentation_id}',
```
Comment (bbezak) [inline networking_generic_switch/devices/netmiko_devices/cumulus.py:126]: been testing it elsewhere, and delete port is incorrect, it shouldn't have segmentation_id:

```
 read_channel: nv unset interface swp1 bridge domain br_default access 71
nv set interface swp1 bridge domain br_default access 72
nv config apply --assume-yes
Error: Invalid parameter: '71'
```

**Review by bbezak — CHANGES_REQUESTED** [review]

File: networking_generic_switch/devices/netmiko_devices/cumulus.py (hunk 1 above)
Comment (bbezak) [reply networking_generic_switch/devices/netmiko_devices/cumulus.py:126]: even if I remove this part of the code, the driver is not changing the switch port vlans (cumulus 5.6), and neutron with debug is not providing any information about port binding using netmiko. Cumulus 4 works fine. Need to test it more.

Comment (bbezak) [reply networking_generic_switch/devices/netmiko_devices/cumulus.py:126]: this one works - https://review.opendev.org/c/openstack/networking-generic-switch/+/907551

Comment (bbezak) [reply networking_generic_switch/devices/netmiko_devices/cumulus.py:126]: remove/applied access vlan correctly:

```nv unset interface swp2 bridge domain br_default access
nv set interface swp2 bridge domain br_default access 71
nv config apply --assume-yes
nv unset interface swp2 bridge domain br_default access

nv set interface swp2 bridge domain br_default access 71
nv config apply --assume-yes```

Comment (markgoddard) [reply networking_generic_switch/devices/netmiko_devices/cumulus.py]: I've fixed up the unit tests upstream and updated the cherry pick in this PR.

Comment (mnasiadka) [reply networking_generic_switch/devices/netmiko_devices/cumulus.py]: Is it time to merge it finally?

**Review by mnasiadka — APPROVED** [review]

**Pull request merged**